vm/src/stdlib/io.rs

```diff
@@ -128,6 +128,10 @@ impl PyStringIORef {
         }
     }
 
+    fn seekable(self, vm: &VirtualMachine) -> PyResult {
+        Ok(vm.ctx.new_bool(true))
+    }
```

Comment:
Minor thing, but functions like these can just return `bool`, some trait magic will work out how to convert it into a PyObject.